recipes/apr-util/all/conanfile.py

```diff
@@ -104,6 +104,8 @@ def requirements(self):
             self.requires("expat/2.2.9")
         if self.options.with_postgresql:
             self.requires("libpq/11.5")
+        if self.settings.os != "Windows":
+            self.requires("libuuid/1.0.3")
```

Comment:
@madebr  just question - is the uuid really needed?
your test_package was generating for me uuid prints if i've removed the uuids, and i am pretty sure that i dont have uuid lib installed on that system.

so the q is, does then apr-util uses own impl to generate uuids or?
if thats the case maybe it should be setting.

Reply:
My system needs it:
Without a `libuuid` requirement of a `uuid` in `system_libs`, the test_packge fails with:
```
/usr/bin/ld: /home/maarten/.conan/data/apr/1.7.0/_/_/package/6af9cc7cb931c5ad942174fd7838eb655717c709/lib/libapr-1.a(rand.o): in function `apr_os_uuid_get':
rand.c:(.text+0xc): undefined reference to `uuid_generate'
collect2: error: ld returned 1 exit status
```
I looked a bit further into this, and it looks like it is `apr` that is needing libuuid instead of apr-util. So I'll move this requirement to `apr`.

Reply:
Or no, how about this.
If somebody really requires uuid support, he'll need to add `libuuid/1.0.3` to his profile.
If, in the future, some application really needs apr uuid support, we can always add this requirement.
I'll add a comment noting this.